interview-questions.md

```diff
@@ -7,60 +7,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer:An instance variable starts with the @ symbol and is set to equal something. The diffence between variables and instance variables is that variables can only be accesed within the local scope and instance variables can be accessed anywhere by calling on them.
 
-  Researched answer:
+  Researched answer:The instance variable ,starts with an @ symbol and found in body of the initailize method, is a variable that can be accessible in any instance method in a particular instance of a class.
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: A block in ruby is a block of code that starts with a do and ends with an end.
 
-  Researched answer:
+  Researched answer:A block in Ruby is a grouping of code that accepts arguements and returns a value and is always passed to a method call.
 
 
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer:Implicit return basically an automatic return thats built in to ruby. It will automatically return something with out actually coding return.
 
-  Researched answer:
+  Researched answer:Every block in ruby returns a value automatically of the last executed code in the block this is because of Implicit return.
 
 
 
 4. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: Ruby is an object oriented program that uses classes and objects.
```

Comment:
I can see where you're going with this answer and I appreciate the attempt to answer with an example. Next time try answering the big 6 questions: Who, What, When, Where, Why, How? That should give you a few sentences of info to elaborate on.

Reply:
ok sounds good thanks